src/Agent/NewRelic/Agent/Core/Api/AgentApi.cs

```diff
@@ -632,7 +632,7 @@ void work()
             {
                 InternalApi.ErrorFingerprintingCallback(callback);
             }
-            TryInvoke(work, apiName, ApiMethod.ErrorFingerprintingCallback);
+            TryInvoke(work, apiName, ApiMethod.ErrorGroupCallback);
```

Comment:
Could we have `Set` in the name, so it's clear what it's doing? This makes it seem like we're invoking the callback here.

Reply:
The spec currently doesn't have the set verb.

Reply:
We discussed this in pairing and decided to go with Set...

Reply:
I plan on updating the API itself (and not just the enum) with a separate PR.